server/services/workAndSkillsPageService.ts

```diff
@@ -73,14 +73,16 @@ export default class WorkAndSkillsPageService {
     let UNACABLastSixMonths = 0
     let UNACABLastMonth = 0
 
-    offenderAttendanceHistory.content.forEach(absence => {
-      if (absence.outcome !== undefined && absence.outcome === 'UNACAB') {
-        UNACABLastSixMonths += 1
-        if (absence.eventDate > oneMonthAgo) {
-          UNACABLastMonth += 1
+    if (offenderAttendanceHistory !== undefined && offenderAttendanceHistory.content.length) {
+      offenderAttendanceHistory.content.forEach(absence => {
+        if (absence.outcome !== undefined && absence.outcome === 'UNACAB') {
```

Comment:
Would be good to have `'UNACAB'` in an `enum` for consistency and readability, especially if there might be more attendance outcome codes used in other places.